sdk/nodejs/appservice/plan.ts

```diff
@@ -235,7 +237,7 @@ export interface PlanState {
     /**
      * The kind of the App Service Plan to create. Possible values are `Windows` (also available as `App`), `Linux`, `elastic` (for Premium Consumption) and `FunctionApp` (for a Consumption Plan). Defaults to `Windows`. Changing this forces a new resource to be created.
```

Comment:
Do you know why the set of values in the comment is less than the set of possible values that you defined?

Reply:
This is actually in the Terraform documentation - we are taking that as is